### ISSUE TITLE
Maintain cursor position after completion

### ISSUE THREAD

**Issue by j6k4m8** [rymate1234/tab-complete-input-vue#1]
Really enjoying using this! Would love the ability to keep cursor position after autocompleting.

Before pressing tab to complete `autoc` to `autocomplete`, the cursor is positioned in the middle of the text:

```
Autoc𝙸 this text
     ^
```

After tab-completing, the cursor hops to the end of the input.
```
Autocomplete this text𝙸
                      ^
```

Curious if this capability exists yet and I'm missing a flag?

**Comment by rymate1234** [comment]
This has been fixed by `1.0.17`

**Comment by j6k4m8** [comment]
Awesome! Looking forward to using.
